Sorted Fixes/sorted_fixes.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import pandas as pd
+import json
+import sys
+import os
+import datetime
+    
+#adjust base_url value if your account is on a different platform
+base_url = "https://api.kennasecurity.com"
+fixes_url = base_url + "/fixes"
+
+headers = {"Accept": "application/json", "X-Risk-Token":os.environ.get("kenna_api_key"), "User-Agent": 'sorted_fixes/1.0.0 (Kenna Security)'}
+
+sorted_fixes = pd.DataFrame()
+
+try:
+    first_page = requests.get(fixes_url, headers=headers).json()
```

Comment:
I would recommend checking for a HTTP 200 response here. Connection Error doesn't cover everything.